start/scheduler.ts

```diff
@@ -1,11 +1,20 @@
 import scheduler from 'adonisjs-scheduler/services/main'
-import { runScrapper } from '../scripts/scrapper.js'
+import { runScrapper as runMenuScrapper } from '../scripts/menu_scrapper.js'
+import { runScrapper as runUsersScrapper } from '../scripts/users_scrapper.js'
 import logger from '@adonisjs/core/services/logger'
 
 scheduler
   .call(() => {
-    logger.info('Running scraper.')
-    void runScrapper()
+    logger.info('Running menu scraper.')
+    void runMenuScrapper()
   })
   .immediate()
-  .everyFifteenMinutes()
+  .everyFiveMinutes()
+
+scheduler
+  .call(() => {
+    logger.info('Running users scraper.')
+    void runUsersScrapper()
```

Comment:
technically it's not really a scrapper, more like a wrapper I guess. But you can leave it (everybody knows what you've meant)